Dockerfile

```diff
@@ -63,6 +63,12 @@ RUN ./scripts/install_full.sh && python ./scripts/trim_sdk.py \
     )" \
  && apk add --virtual .rundeps $runDeps
 
+# update vulnerabilities
+RUN apk upgrade openssl
+RUN apk upgrade binutils --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
```

Comment:
This is not a universal solution as they are only specific to certain versions of Alpine Linux image. Alpine Linux will include these fixes in its new docker image. We also automatically pick up the latest Alpine Linux image: https://github.com/Azure/azure-cli/blob/5f52c83e218f3ddc4d933bd99a88dab9e1864d81/Dockerfile#L8